ui/src/submissions/common/components/DateField.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { DatePicker } from 'antd';
+import moment from 'moment';
+
+import withFormItem from '../withFormItem';
+
+class DateField extends Component {
+  constructor(props) {
+    super(props);
+    this.onBlur = this.onBlur.bind(this);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onBlur() {
+    const { form, name } = this.props;
+    form.setFieldTouched(name, true);
+  }
+
+  onChange(date, dateString) {
+    const { form, name } = this.props;
+    form.setFieldValue(name, dateString);
+  }
+
+  render() {
+    const { value, ...otherProps } = this.props;
+    return (
+      <DatePicker
+        className="w-100"
+        {...otherProps}
+        data-test-type="date-picker"
+        defaultValue={value && moment(value)}
```

Comment:
don't we need a specific format?

Reply:
`DatePicker` has default format which is `YYYY-MM-DD`